src/app/core/services/navigation/nav-tree.service.ts

```diff
@@ -21,13 +20,16 @@ export interface NavigationNeighbors {
 export abstract class NavTreeService<ContentT extends RoutedContentInfo> {
 
   private reloadTrigger = new Subject<void>();
+  protected reload$ = merge(
+    this.reloadTrigger,
+    this.currentContent.reload$,
+  );
 
   private content$ = this.currentContent.content$.pipe( // only keep those of interest for the current nav tree
```

Comment:
So, if I understand it well, you've put the refresh on the children fetching only. So they will only be a refresh in the children data, not in the root one, right?
From what I see, that's the case. Of course, as there is still the former "trick" which push in "details" the title, score, ... these are updated, but not because of your changes.
For instance, if I remove myself from a group, the menu is not updated accordingly.
The idea was to have an actual force refresh of the whole menu to make sure any info is up to date.

Reply:
I don't see the remark addressed. 
Ok, probably the consequences are probably fixed with emitting empty array for element which do not have children...  
But that still strange to me not to emit a new content (so no start fetching the main content) when there is such a refresh. As a consequence, actually the content will not be refreshed immediately but, if I understand well, will be refreshed as a consequence (so after) of the children change.